src/main/java/com/monov/cloud/gateway/service/CourseGatewayService.java

```diff
@@ -0,0 +1,47 @@
+package com.monov.cloud.gateway.service;
+
+import com.monov.cloud.gateway.dto.Course;
+import com.monov.cloud.gateway.dto.StudentOrCourseIds;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+@Service
+public class CourseGatewayService {
+    public static final String COURSE_SERVICE_URL = "http://localhost:9002/courses";
```

Comment:
Extract this into a config file (.properties or .yaml) and use the @Value annotation to inject it here.

Reply:
Fixes will be applied in DTOs branch's PR